### PR TITLE
Meetup on software testing

### PR DESCRIPTION
**Pull request by DaanVanVugt**
I also reordered the old meetups to be in reverse chronological order.

Still needs an eventbrite page and a review of the agenda

### CODE REVIEW

**Review by c-martinez — APPROVED** [review]
Looks good -- do you want to add the Eventbrite event before merging? Or can it be merged already to start advertising and add the Eventbrite page afterwards.

**Review by tpronk — APPROVED** [review]

**Comment by DaanVanVugt** [comment]
Dan has suggested some more detail on the event location, which has been added. @c-martinez could you copy-paste this into an eventbrite page? Let's say maximum 40 attendees for now.

File: content/events/2023-06-15-meetup.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+
+## Location
+[Centre for Digital Scholarship, Leiden University Library, Witte Singel 27, Leiden](https://goo.gl/maps/JgG4bmXW3LtMxDZ36)
+The CDS is on the ground floor of the Leiden University Library, directly straight across the main room from the entrance.
+There will be a neon sign above the door that says Centre for Digital Scholarship.
+
+
+## Registration
+The registration is free. Please register via this [Eventbrite page](https://www.eventbrite.co.uk/e/).
```
Comment (c-martinez) [inline content/events/2023-06-15-meetup.md:23]:
```suggestion
The registration is free. Please register via this [Eventbrite page](https://www.eventbrite.com/e/nl-rse-automated-software-testing-tickets-633227900697).
```

Comment (c-martinez) [reply content/events/2023-06-15-meetup.md:23]: @DaanVanVugt -- ready to go!

**Comment by DaanVanVugt** [comment]
* introduction to
* target audience

File: content/events/2023-06-15-meetup.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+---
+layout: event
+image: '/img/meetup-header-with-logo.jpg'
+title: "Automated software testing"
```
Comment (c-martinez) [inline content/events/2023-06-15-meetup.md:4]:
```suggestion
title: "Automated software testing for dummies"
```

**Comment by tpronk** [comment]
Summary of feedback on current event description (discussed in a core-team meeting):
* Make the target audience clear. What prerequisites do they need?
* Make it clear in the title this is for beginners: "Automated Testing for Dummies" or "Introduction into Automated Testing"

**Comment by DaanVanVugt** [comment]
I updated the PR to reflect our discussions. Perhaps it would be good to transplant some of this to the eventbrite page? The registration form looks good!

**Review by darudmann — APPROVED** [review]

**Pull request merged**